### ISSUE TITLE
Adding time statistics

### ISSUE THREAD

**Issue by stefanesterer** [zzorba/ArkhamCards#286]
Hi!

First of all thx for all the great work you are doing with this app! This can't be repeated enough :)

I'm using the app a lot and something that I and my fellow players are missing is time information about when we started a campagin, finished a scenario and so on. 

In the backuped campaign data I found an entry "lastUpdated" on the campaign level, but nothing of the sort on the scenarios themselves. My suggestion would be to add a started and finished timestamp on the scenario level. This would allow the user to know when they started a campaign (which is not possible right now as far as I know) and also to find out afterwards how long it took to finish a scenario.

Would it be fine if I try to implement this and then suggest a PR? My first step would be only to add the data, no changes in the UI.

Pls let me know if this is something you are interested in and if you would like to get contributions like that..

yours
stefan

**Comment by zzorba** [comment]
Yes I'd be open to an addition like this, just make sure the date is 'optional' in the typescript schema, since old ones won't have it.
